person.rb

```diff
@@ -11,9 +11,10 @@ def initialize(age, name, parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
-  attr_reader :id
+  attr_reader :id, :rentals
   attr_accessor :name, :age
 
```

Comment:
- [ OPTIONAL ] Kindly I suggest you use your attr_reader and attr_accessor at the top of your class just after the name of the class for conventional reasons and to make them accessible inside your all class :+1: